frontend/src/pages/acceleratorProfiles/screens/manage/tolerations/TolerationRow.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { ActionsColumn, Td, Tr } from '@patternfly/react-table';
+import { PodToleration } from '~/types';
+
+type TolerationRowProps = {
+  toleration: PodToleration;
+  onDelete: (toleration: PodToleration) => void;
+  onEdit: (toleration: PodToleration) => void;
+};
+
+export const TolerationRow: React.FC<TolerationRowProps> = ({ toleration, onEdit, onDelete }) => {
+  const formatValue = (value: string | undefined) => value || '-';
+  const formatSeconds = (seconds: number | undefined) => (seconds ? `${seconds} seconds(s)` : '-');
+
+  return (
+    <Tr>
+      <Td dataLabel="Operator">{formatValue(toleration.operator)}</Td>
+      <Td dataLabel="Key">{formatValue(toleration.key)}</Td>
+      <Td dataLabel="Value">{formatValue(toleration.value)}</Td>
+      <Td dataLabel="Effect">{formatValue(toleration.effect)}</Td>
+      <Td dataLabel="Toleration Seconds">{formatSeconds(toleration.tolerationSeconds)}</Td>
+      <Td isActionCell>
+        <ActionsColumn
```

Comment:
```suggestion
        <ActionsColumn
         menuAppendTo="parent"
```

Reply:
that is standard behavior for any dropdown, action menu, etc... the reason its weird here is that it is at the bottom. the solution is for the user to scroll a bit to see the delete. it is a bit weird here because you cant scroll enough to see the full menu... but setting that to parent causes the element to go over the footer like a modal would. it would be better to just have the menu go up instead of down
![Screenshot 2023-11-03 at 3 19 53 PM](https://github.com/opendatahub-io/odh-dashboard/assets/12587674/a1e7782b-0a7f-4bec-ac62-a914d8788499)

Reply:
I think it looks off. @yannnz what would you rather see. the menu covering the footer, or the menu opening above the button

Reply:
I vote for the menu opening above the button. The same reason as you suggested @Gkrumbach07. cc: @dpanshug